zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/LearnerHandler.java

```diff
@@ -505,34 +510,35 @@ public void run() {
             // startForwarding() will be called in all cases
             boolean needSnap = syncFollower(peerLastZxid, learnerMaster);
 
+            boolean exemptFromThrottle = getLearnerType() != LearnerType.OBSERVER;
```

Comment:
I think it'll be good to add a comment here what exempt means (sync throttler will not throttle if it's true) and why we need it (IIUC, we want the quorum servers to always sync and ignore throttling because it's important to keep the state of quorum serves up to date with leader).

Reply:
done